Makefile

```diff
@@ -12,52 +12,58 @@
 
 EXTENSION-LIBRARY-FOLDER-NAME = campaignclassic
 
-clean:
-	  (./code/gradlew -p code clean)
+init:
```

Comment:
We may not need githook here.

Reply:
I have added the githook folder to format code before every commit